packages/block-library/src/file/edit.js

```diff
@@ -130,7 +130,7 @@ function FileEdit( {
 				fileName: newMedia.title,
 				textLinkHref: newMedia.url,
 				id: newMedia.id,
-				displayPreview: isPdf ? true : undefined,
+				displayPreview: isPdf ? false : undefined,
```

Comment:
Thanks for working on this.

I think it'd be fine to remove this line to indicate that the preview should never display. In my testing it works the same as setting it to `false`.

Similarly, the `previewHeight` line below doesn't seem to be needed and can be removed. The block attribute has a default value of `600` already, so that line doesn't do anything (if it's `undefined` it'll be `600`, if it's `600` it'll be `600`).

Reply:
Actually, I suppose `previewHeight` attribute on the line below will reset the height to 600 if a new file is uploaded and the old one had a different height. Not sure if that's a good thing, but maybe leave it as it is.

Reply:
Has this been committed to core? Wondering how I can test this in my dev environment, as I don't seem to have this file path in my installation.

Reply:
It has not been merged into the plugin yet. If you would like to test the change, the [Getting Started page](https://developer.wordpress.org/block-editor/contributors/code/getting-started-with-code-contribution/) of the handbook should help.

Reply:
@stevebab You can also download the a compiled version of gutenberg (as a zip) that includes this change; just follow the video at https://github.com/WordPress/gutenberg/issues/40483#issuecomment-1104593308 and then put it into your plugins directory :)

Reply:
you can find the zip file specifically at https://github.com/WordPress/gutenberg/actions/runs/2025241537